code/modules/events/abductor.dm

```diff
@@ -1,12 +1,12 @@
 /datum/event/abductor
 
 /datum/event/abductor/start()
-	//spawn abductor team
-	processing = 0 //so it won't fire again in next tick
+	INVOKE_ASYNC(src, /datum/event/abductor.proc/try_makeAbductorTeam)
```

Comment:
```suggestion
	INVOKE_ASYNC(src, .proc/try_makeAbductorTeam)
```
Pretty sure you can use a relative ref to the proc here, but not 100%. Please try for this one, and if you can, update the rest please.

Reply:
Yep that seems to work. Still magic to me at times how byond determines when a relative path works and when it does not